docs/changelog.rst

```diff
@@ -4,6 +4,8 @@ Changelog
 
 * :bug:`3406` Sushiswap events will now be taken into account during the profit and loss report.
 * :bug:`3407` Users will now see correctly reported their amount of xSUSHI.
+* :bug:`3418` Users importing data from crypto.com will now have a better experience when dust is not credited.
+* :bug:`3056` Users will now be better notified when an error occurred while importing information from crypto.com on credit/debit events with special cases.
```

Comment:
you are talking about better experience and better notifications here. But is that what you are doing? Doesn't your change fix the bug, thus making import actually work when doing those things?

There is a big difference between "improving experience" and "making it work"

Reply:
Without this fix the csv import didn't get to finish and no data was imported. Now what I do, in the case of data that we expect and is missing, we notify the user about it and continue with the import 

in #3418 there was an amount that is "burned" but as how we know crypto.com works we expect two events, so we were expecting two events and only getting one. In this case we ignore this row and notify the user

Reply:
Also another thing is that credited/debited events always go on pairs and the timestamp is the same (this is the logic that was first implemented and what happens most of the time) but reading the #3056 csv file I found entries that were off by one second and this breaks things (in +1000 lines only happened twice).

That's why I talk about improving experience

Reply:
> Without this fix the csv import didn't get to finish and no data was imported

So why not mention that CSV import for crypto.com is actually working again. That's the only thing you need to say.